tests/Feature/Networks/APIv2NetworkTest.php

```diff
@@ -81,6 +83,10 @@ public function testGet(): void {
      */
     #[DataProvider('providerGroupsParameters')]
     public function testListGroups($getNextEvent, $getDetails): void {
+        // Create a test user with admin privileges and authenticate
+        $user = $this->createUserWithToken(Role::ADMINISTRATOR);
+        $this->actingAs($user);
```

Comment:
Should `createUserWithToken` just call `actingAs`? I guess maybe we just sometimes want the user and don't want them to be "logged in".

Reply:
These tests do need to be audited. In some tests it would make sense to make actual HTTP requests to login, other tests we'd just want to have a user to use so the fast login method would be best, and then there are some tests where we just use the user api token for requests while other times we use the same user to do stuff.

The tests really need to be more divided than just directory paths.